src/main/resources/db/changelog/ddl/0026_update_aed_summary_view.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="vishal.gupta@valtech.com" id="0026_update_aed_summary_view - Adds id to AED summary view" context="ddl">
```

Comment:
You know you want to make this 0027! https://github.com/cabinetoffice/eip-ero-print-api/pull/237 😆 😉

Reply:
Technically this PR was raised before #237  😜 but If you want I can wait for your PR to be merged first

Reply:
I don't mind. It depends on which one gets merged first 😄

Reply:
I've renamed mine to 27 👍